includes/common.php

```diff
@@ -994,11 +994,11 @@ function load_os(&$device)
 
     // Set type to a predefined type for the OS if it's not already set
     $loaded_os_type = Config::get("os.{$device['os']}.type");
-    if ((! isset($device['attribs']['override_device_type']) && $device['attribs']['override_device_type'] != 1) && array_key_exists('type', $device) && $loaded_os_type != $device['type']) {
-        log_event('Device type changed ' . $device['type'] . ' => ' . $loaded_os_type, $device, 'system', 3);
```

Comment:
as i remember will log event be printed in Eventlog, why did you remove this?
A device type change is a noteable information for user, in my eyes :-)

Reply:
it's moved to the device observer